src/plasoscaffolder/bll/mappings/formatter_init_mapping.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+"""Class representing the mapper for the formatter init files."""
+from plasoscaffolder.bll.mappings import base_mapping_helper
+from plasoscaffolder.bll.mappings import base_sqliteplugin_mapping
+from plasoscaffolder.model import init_data_model
+
+
+class FormatterInitMapping(
+    base_sqliteplugin_mapping.BaseSQLitePluginMapper):
+  """Class representing the formatter init mapper."""
+
+  _FORMATTER_INIT_TEMPLATE = 'formatter_init_template.jinja2'
+
+  def __init__(self, mapping_helper: base_mapping_helper.BaseMappingHelper()):
```

Comment:
Python allows you to deviate from pure abstract classes which allows you to move `__init__` into BaseSQLitePluginMapper as well.